### DISCUSSION TITLE
How to make an s3 bucket path that doesn't exist still load

### DISCUSSION THREAD

**Discussion by mlordi** [oneduo/nova-file-manager#447] [Q&A]
I have a scenario where I have projects that might not have any documents loaded to an s3 bucket yet. I have a button on the project page that will launch the file manager with the path predefined (i.e. /nova-file-manager?path=%5Cprojects%2F1629&disk=s3). I would like to have this still load the file manager but instead of just seeing the blue loading symbol forever that it would fail gracefully and just show a message that no files currently exist. Any ideas?

**Comment by Rezrazi** [comment]
I don't believe we support at this moment, we rely on Laravel's file system implementation to list the files in the tool, if the path doesn't exist we expect the storage adapter to fail.

Happy to consider a PR.
